dataactcore/utils/jobQueue.py

```diff
@@ -37,6 +71,8 @@ def __init__(self, job_queue_url="localhost"):
         self.jobQueue.config_from_object('celeryconfig')
 
         self.enqueue = self.jobQueue.task(name='jobQueue.enqueue')(enqueue)
+        self.generate_f_file = self.jobQueue.task(
```

Comment:
It's a good question, and I'd argue it's rather messy right now. The `JobQueue` class creates a `Celery` connection in its constructor and then sets some convenience functions for using the two asynchronous functions (`enqueue` and now `generate_f_file`).  Celery [suggests](http://docs.celeryproject.org/en/latest/getting-started/first-steps-with-celery.html#application) tasks be set up like:

```python
app = Celery(...)

@app.task
def my_function():
   ...
```
I'm not too sure why we're currently setting all that up in a constructor. Maybe we needed to delay to run time for some reason?

This PR takes one step to simplifying that by moving the actual logic of those functions out of the initializer, but we can probably go further. I'll take a stab.